src/solver/modeler/dataSeries/include/antares/solver/modeler/dataSeries/dataSeriesRepo.h

```diff
@@ -0,0 +1,24 @@
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include "dataSeries.h"
+
+namespace Antares::Solver::Modeler::DataSeries
+{
+
+class DataSeriesRepository
+{
+public:
+    void addDataSeries(std::unique_ptr<IDataSeries> dataSeries);
```

Comment:
About `void addDataSeries(...)` : forces to **std::move** a unique pointer defined outside. 
So the client **IDataSeries** gets empty after this call.